src/Symfony/Component/Config/Definition/EnumNode.php

```diff
@@ -43,12 +51,35 @@ public function getValues()
         return $this->values;
     }
 
+    /**
+     * @internal
+     */
+    public function getPermissibleValues(string $separator): string
+    {
+        return implode($separator, array_unique(array_map(static function (mixed $value): string {
+            if (!$value instanceof \UnitEnum) {
+                return json_encode($value);
+            }
+
+            return '\\'.ltrim(var_export($value, true), '\\');
```

Comment:
it'd suggest removing the leading backslash

Reply:
this is used to generate the phpdoc on the generated classes, where we need this to be fully qualified.

Reply:
It looks like `ConfigBuilderGenerator` uses its own logic. To me, this method is only used to compute messages.

Reply:
Yes, it's used only in exception messages and config reference dumps, I removed it. `ltrim` stays because PHP 8.2 var_export adds a leading backslash.